ext/osx/pxp-agent.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+                <key>PATH</key>
+                <string>/opt/puppetlabs/puppet/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+                <key>LANG</key>
+                <string>en_US.UTF-8</string>
+        </dict>
+        <key>Label</key>
+        <string>pxp-agent</string>
```

Comment:
FWIW my $.02 is: it sounds like we don't have the work scoped for this change, and also consistency is nice to have. So for now, I'd leave this one unqualified (i.e. just `pxp-agent`), and create a ticket (or tickets) for updating all the agent services to fully-qualified names and making whatever related changes are needed in, say, the enterprise modules managing these services (and other modules?).

Thoughts?